### ISSUE TITLE
publish latest version to npm

### ISSUE THREAD

**Issue by faalsh** [weixsong/elasticlunr.js#60]
Currently the latest version in nom repository is 0.9.5 and in github is 0.9.6

Could you please publish the latest version to npm?

**Comment by jlintag** [comment]
This would be very helpful; current documentation points to field search but that functionality isn't implemented in 0.9.5.

**Comment by drschwabe** [comment]
For anyone who needs a quick copy/paste solution for their package.json in the meantime: 
```json
"dependencies": {
   "elasticlunr": "git://github.com/weixsong/elasticlunr.js.git#1995e0f73a395c620b2fb597f8ac3ec61b831ffa"
}
```
(replace hash with latest commit if necessary)

**Comment by marksyzm** [comment]
Yeah there's a breaking change there between the two! I just noticed that the `npm` version doesn't give you the appended doc property to the search results:
https://runkit.com/marksyzm/5ae20ccaac30a60013f42077
